src/session/sessions.ts

```diff
@@ -0,0 +1,284 @@
+//#region > Imports
+//#PACKAGE "graphql"
+//## npm install "graphql"@14.6.0
+// Contains the interface for gql queries, mutations and subscriptions
+import { DocumentNode } from "graphql";
+//#PACKAGE "js-cookie"
+//## npm install "js-cookie"@2.2.1
+// A simple, lightweight JavaScript API for handling browser cookies
+import Cookies from "js-cookie";
+
+//> Session
+// Contains the base session
+import Session from "./index";
+//> Templates
+// Contains the main template for the sessions
+import { IMainTemplate } from "../templates/index";
+// Contains the SNEK template
+import SnekTemplate from "../templates/snek/index";
+//> Tasks
```

Comment:
I'd add a new line infront of `//> Tasks` as you did with `//> Session` above.

Reply:
Te newline above separates the external and internal imports.